site/content/blog/kapp-rebase-rules.md

```diff
@@ -0,0 +1,106 @@
+## Kapp Rebase Rules
```

Comment:
We need details like title, slug, author, etc for blogs. (Maybe you are planning to add it before merging but thought of mentioning it anyway 😄 )

`Sample`
```md
---
title: "kapp rebase rules"
slug: kapp-rebase-rules
date: 2022-04-26
author: Joe Kimmel
excerpt: "Use kapp rebase rules for preserving some fields in a resource on the cluster"
image: /img/logo.svg
tags: ['kapp', 'rebase-rules']
---
```

Reply:
😅 thanks!